seacrowd/sea_datasets/yunshan_cup_2020/yunshan_cup_2020.py

```diff
@@ -0,0 +1,164 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{DBLP:journals/corr/abs-2204-02658,
+ author    = {Yingwen Fu and
+              Jinyi Chen and
+              Nankai Lin and
+              Xixuan Huang and
+              Xin Ying Qiu and
+              Shengyi Jiang},
+ title     = {Yunshan Cup 2020: Overview of the Part-of-Speech Tagging Task for
+              Low-resourced Languages},
+ journal   = {CoRR},
+ volume    = {abs/2204.02658},
+ year      = {2022},
+ url       = {https://doi.org/10.48550/arXiv.2204.02658},
+ doi       = {10.48550/arXiv.2204.02658},
+ eprinttype = {arXiv},
+ eprint    = {2204.02658},
+ timestamp = {Tue, 12 Apr 2022 18:42:14 +0200},
+ biburl    = {https://dblp.org/rec/journals/corr/abs-2204-02658.bib},
+ bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+"""
+
+_DATASETNAME = "yunshan_cup_2020"
+
+_DESCRIPTION = """\
+Lao POS dataset containing 11,000 sentences was released as part of Yunshan-Cup-2020 evaluation track.
+"""
+
+_HOMEPAGE = "https://github.com/GKLMIP/Yunshan-Cup-2020"
+
+_LICENSE = Licenses.UNKNOWN.value  # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_URLS = {
+    "train": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/train.txt",
+    "val": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/dev.txt",
+    "test": "https://raw.githubusercontent.com/GKLMIP/Yunshan-Cup-2020/main/test.txt",
+}
+_SUPPORTED_TASKS = [Tasks.POS_TAGGING]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class YunshanCup2020(datasets.GeneratorBasedBuilder):
```

Comment:
Add `Dataset` Suffix at class name
```suggestion
class YunshanCup2020(datasets.GeneratorBasedBuilder):
```